libraries/RC_Channel/RC_Channel.h

```diff
@@ -330,6 +333,8 @@ class RC_Channel {
 
     // pwm is stored here
     int16_t     radio_in;
+    // previous radio_in, used to ignore small changes
```

Comment:
we should specify that "previous_radio_in" is only used by radio_input_changed().

Reply:
renamed